### PR TITLE
Update NetUtil.java

### PR DESCRIPTION
**Pull request by xiaoqiang-github-123**
fix bug: Ipv6环境下，io.seata.common.util.NetUtil的toInetSocketAddress方法没办法正确解析ipv6格式的地址 

issue: https://github.com/seata/seata/issues/5607

- [ ] I have registered the PR [changes](../changes).

### CODE REVIEW

**Review by a364176773 — COMMENTED** [review]
Can you add relevant test cases?

File: common/src/main/java/io/seata/common/util/NetUtil.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ public static String toStringAddress(InetSocketAddress address) {
      * @return the inet socket address
      */
     public static InetSocketAddress toInetSocketAddress(String address) {
-        int i = address.indexOf(':');
+        int i = address.lastIndexOf(':');
```
Comment (slievrly) [inline common/src/main/java/io/seata/common/util/NetUtil.java:86]: It seems to me that NetUtil's IPV6 support is not just about this.

`LOCALHOST`  `ANY_HOST`  `IP_PATTERN`  `toLong`

**Comment by leizhiyuan** [comment]
https://github.com/seata/seata/pull/5902 

it seems others had fixed this，please try the new version

**Review by slievrly — COMMENTED** [review]
Since there is no feedback for a long time, I will close this PR. If you have any questions, please open it again.

**Pull request closed without merging**